### DISCUSSION TITLE
Can we include Pose detection

### DISCUSSION THREAD

**Discussion by god2b** [blakeblackshear/frigate#1504] [Ideas]
Hi All,

A bit of back story, i have a very stubborn elderly aunt who wishes to continue living at home by herself. She is getting frail and has fallen a couple of times already, and on one occasion was unable to get up and seek help for over 8 hours. She has one of those panic button thingys, but had decided she was only going a short distance and didn't take it with her.

I had come across pose detection https://towardsdatascience.com/fall-detection-using-pose-estimation-a8f7fd77081d and here https://medium.com/beyondminds/an-overview-of-human-pose-estimation-with-deep-learning-d49eb656739b and had looked to see if there was anyway to integrate into home assistant, but i have not come across anything. 

The hope was to add a few cameras in her home to monitor for trips or falls, and immediately notify family members that an incident has occured. Rather than someone being able to watch the cameras and compromise my aunts privacy, the pose detection could monitor for the fall event without anyone invading my aunts privacy. 

I guess the same tech could be used in a playground or around a swimming pool area, to watch for kids being possibly injured or someone falling into a pool?

Hopefully this is something we can include into frigate, i know it my new favourite tool at the moment and has real life benefits, but including the pose detection in as well could have life changing/saving effects also.

Regards
Stephen

**Comment by scooper1** [comment]
it would be very difficult to put cameras in all places required to detect a fall  a no-movement alarm might be a better idea 
FYI  the panic button thingy also has a fall detection version and I think in my personal opinion  for many reasons you be better off getting one of those if you can get her to wear it

**Reply by god2b** [reply]
Hi scooper1, thanks for the feedback.

She currently has the panic button with fall detection, but between getting her to charge the unit and using it also is a challenge, she is in early stages of alzheimer's and do things she kinda should be doing can be a struggle. My mindset was something that isn't relient on her own actions, and the pose detection i feel is a winner. Also iv been told the unit was around £200 and it has a monthly sub of around £20 per month, thats alot of money.

In terms of a no-movement alarm, pir sensors were my first thought, but when she sits down to watch tv or leaves a room the data is difficult to determine, same as a fall.

Making use of the cameras, for movements, zones like fridge or at the kitchen table could tell us a lot more of the daily happens to make sure movement is happening without intruding or breaching her privacy. 

Look, if its something that could be included as an option down the line, that would be amazing, its something i personally would love to see included. Im sure there are alot of people out there whom would love to be reassured their loved ones well.

**Comment by scooper1** [comment]
I understand  - my own mother has just passed from effects of dementia after 20 years of it - and i lived at home
I would go via your own private fall alarm I am sure they will be available plus your own movement sensors all integrated by hassio
The problem is as I see it if she falls and your cameras don't detect it the family will be at you 
dementia causes many family problems ..... 

it very unlikely that object detection will work reliably coping with 10 jumpers on to nothing etc ... 

power monitors are a good idea and Zigbee door open sensors   

a Fitbit might work   - these are getting better all the time  

its is a minefield to deal with   

door access systems and cameras to check on carers is a better thing 

looking after room temperature is also good  

what i am trying to say is the falls are a minor issue that looks very big at the moment  

not eating and drinking  getting too hot too cold letting wrong people in etc is the real problem current tech can help with

sorry 20 years of dealing with the problem have been difficult

**Reply by scooper1** [reply]
current detection system in the house is not reliable enough for medical use often seeing people as dogs etc let alone being able to detect someone fallen against a wall 

my mother would have just used cameras as coat hooks if she got the idea that they were looking at here

**Comment by krasatos** [comment]
Hey @god2b, how did you end up handling this issue, i am in a similar situation.
Any ideas?
TiA

**Comment by secnigma** [comment]
Pardon for the necrobumping, but just posting here, to help the future netizens, who are worried about their loved ones.


Fall detection can now be achieved by 24Ghz mmwave sensors, such as this [Seed Studio MR24FDB1](https://wiki.seeedstudio.com/Radar_MR24FDB1) or this [Seeed Studio MR60FDA2 60GHz](https://esphome.io/components/seeed_mr60fda2) fall detection sensors.

If [fitted properly](https://wiki.seeedstudio.com/getting_started_with_mr60fda2_mmwave_kit/), these mmwave sensors can potentially achieve better, cheaper and efficient fall detection, without frigate or any other object detection models.
